libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -248,6 +263,10 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
   private listenForIonModalWillDismiss() {
     if (this.ionModalElement) {
       this.ionModalElement.addEventListener('ionModalWillDismiss', () => {
+        this.renderer.removeClass(
```

Comment:
Consider removing class on the 'ionModalDidDismiss' event instead, to avoid disappearing scrollbar while the modal is closing.